src/cs50/__init__.py

```diff
@@ -19,8 +41,8 @@
     # Replace Flask's logger
     from . import flask
 
-    # Wrap SQLAlchemy
-    from .sql import SQL
+    # Lazily load CS50.SQL
+    sys.meta_path.append(CustomImporter())
```

Comment:
At this point `cs50.flask` has already imported `cs50.sql`, so there's not much to gain